recipes/perl-list-moreutils-xs/meta.yaml

```
@@ -11,8 +11,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 2
-
+  number: 3
+  run_exports: '{{ pin_subpackage(name,  max_pin="x.x") }}'
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)